modules/fluid_properties/src/fluidproperties/SalineMoltenSaltFluidProperties.C

```diff
@@ -0,0 +1,133 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+//
+#include "SalineMoltenSaltFluidProperties.h"
+
+registerMooseObject("FluidPropertiesApp", SalineMoltenSaltFluidProperties);
+
+InputParameters
+SalineMoltenSaltFluidProperties::validParams()
+{
+  InputParameters params = SinglePhaseFluidProperties::validParams();
+  params.addClassDescription("Molten salt fluid properties using Saline");
+  params.addRequiredParam<std::string>(
+      "comp_name", "The name of the components in the salt (e.g., 'LiF-NaF-KF')");
+  params.addRequiredParam<std::string>(
+      "comp_val", "The mole fraction of each salt component (e.g., '0.5-0.1-0.4)");
```

Comment:
```suggestion
  params.addRequiredParam<std::string>(
      "comp_name", "The name of the components in the salt (e.g., 'LiF-NaF-KF')");
  params.addRequiredParam<std::string>(
      "comp_val", "The mole fraction of each salt component (e.g., '0.5-0.1-0.4)");
```

I'd much rather see those as type `std::vector<std::string>` and `std::vector<Real>`. I understand that this is the format saline expects, but it is unexpected for MOOSE users. You can easily `Mosoe::stringify` these vectors with a custom delimiter ('-`) to generate teh Saline compatible argument string.

Reply:
I'd rather be the one to resolve

Reply:
done. Please resolve!

Reply:
resolving